tooling/minimize/src/ty.rs

```diff
@@ -11,15 +11,16 @@ impl<'tcx> Ctxt<'tcx> {
             let size = translate_size(layout.size());
             let align = translate_align(layout.align().abi);
             let layout = LayoutStrategy::Sized(size, align);
-            let nonfreeze_bytes = self.nonfreeze_bytes_in_sized_ty(ty, span);
+            // The ranges are sorted in descending order, but we want them to be in ascending order.
```

Comment:
huh, that's surprising. what is the example where they are sorted in descending order? I think the list should be constructed the right way around, so that we don't have to reverse it all the time :upside_down_face: